pkg/computenodeopenstack/configmap.go

```diff
@@ -0,0 +1,29 @@
+package computenodeopenstack
+
+import (
+        util "github.com/openstack-k8s-operators/lib-common/pkg/util"
+	computenodev1alpha1 "github.com/openstack-k8s-operators/compute-node-operator/pkg/apis/computenode/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// ScriptsConfigMap - scripts config map
+func ScriptsConfigMap(cr *computenodev1alpha1.ComputeNodeOpenStack, cmName string) *corev1.ConfigMap {
```

Comment:
this seems to be generic based on the file/function name, but it is just for the scale script, perhaps it needs renaming